### PR TITLE
LPS-197473 Save button has different dimensions than delete button

### PR DESCRIPTION
**Pull request by Artartur**
Related Issue: [LPS-197473](https://liferay.atlassian.net/browse/LPS-197473)

### CODE REVIEW

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectRelationshipDetails.scss [hunk 1]
```diff
@@ -33,7 +33,7 @@
 		&-save-button {
 			align-items: center;
 			display: flex;
-			height: 32px;
+			height: 40px;
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectRelationshipDetails.scss:36]: Hi @Artartur, the correct height for the button is indeed 32px, to follow the same pattern applied in `RightSidebarObjectFieldDetails`. The problem here is that we are missing the class name `lfr-objects__model-builder-right-sidebar-object-relationship-title-delete-button` in the trash button inside `RightSidebarObjectRelationshipDetails`.

I also noticed we have a problem with the class name for the save button inside `RightSidebarObjectDefinitionDetails`. It is missing the word title before "save-button", can you please fix that too?

Thanks!

**Review by naraandrade — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectRelationshipDetails.scss (hunk 1 above)
Comment (Artartur) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectRelationshipDetails.scss]: Sure, done! Thank you for the review.

**Comment by Artartur** [comment]
ci:test:sf

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:forward:force

**Comment by naraandrade** [comment]
ci:forward:force

**Pull request closed without merging**